mico-core/src/main/java/io/github/ust/mico/core/resource/DeploymentResource.java

```diff
@@ -126,10 +126,10 @@
             // TODO Check if image for the requested version is already in docker registry -> no build required
```

Comment:
Remove TODO (according issue is on hold)

Reply:
Done.